src/modules/ekf2/ekf2_main.cpp

```diff
@@ -980,6 +990,8 @@ void Ekf2::run()
 				_gps_state[1].lat = gps.lat;
 				_gps_state[1].lon = gps.lon;
 				_gps_state[1].alt = gps.alt;
+				_gps_state[1].yaw_offset = gps_yaw_offset;
```

Comment:
yaw (gps.heading) missing?

Reply:
Will fix. Yaw not supported by GPS blending at the moment but will be adding support in a subsequent PR that enables a yaw output and non yaw output receiver to be used together.